### DISCUSSION TITLE
🎉 Let's get to know each other!

### DISCUSSION THREAD

**Discussion by aycabas** [microsoft/hack-together#31] [General]
Hi everyone, we are so excited to kickoff Hack Together on March 1st! Let's use this discussion to introduce ourselves and get to know each other. 🥳

My name is Ayça Baş, I'm a Senior Cloud Developer Advocate at Microsoft 365 Advocacy and I love developing apps with Microsoft Graph. I'm here to help, hack and learn. So excited to get to know you all. 💜

**Comment by waldekmastykarz** [comment]
Hey folks! I'm Waldek and I'm a Senior Developer Advocate at Microsoft, focusing on Microsoft 365. Please, don't hesitate to reach out if you've got any questions about building apps for Microsoft 365 and using Microsoft Graph. Looking forward to meeting you all!

**Reply by zawhtut** [reply]
Thanks for the event. I also want to double check whether you find my name `Zaw Htut` in the registration list?

**Comment by RabebOthmani** [comment]
Hello everyone! My name is Rabeb, I'm a product manager with the Microsoft Graph team focusing on developer experience. 
I'm looking forward to meeting you all! Feel free to reach out if you have any questions, we are here to help :)

**Comment by md-qaiser** [comment]
Hello Everyone, I am a fresh graduate in Computer Science, Looking to enhance my career in .Net development, I am a beginner in this field, having strong knowledge of c#. Open to joining any Team, Together we can build a robust application to conquer this event, Feel free to reach out. Many Thanks !!

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @md-qaiser! Have you already checked out [our resources](https://github.com/microsoft/hack-together#tips--tricks) to get inspiration about what you'd like to hack?

**Comment by andrueastman** [comment]
Hello everyone! My name is Andrew, and I'm a software engineer with the Microsoft Graph SDKs team.
Looking forward to meeting you all and feel free to reach out in case of any questions.
Thanks!

**Comment by fabianwilliams** [comment]
Hi All 👏 My name is Fabian, I am a Senior Product Manager here at Microsoft on the [Microsoft Graph](https://learn.microsoft.com/en-us/graph/overview) Customer and Partner Experience (CPx) team. I am also the Product Lead on the[ MVP Program](https://mvp.microsoft.com/) for the Microsoft Graph. I look forward to seeing and working with you all in the upcoming days. Reach out if you have any questions or feedback. Cheers.

**Reply by Waraemon** [reply]
MVP Program sounds great, added Microsoft MVP to my purpose list <3

**Reply by fabianwilliams** [reply]
Ill be looking out for you and your work contributions!

**Comment by gychin** [comment]
Hello, I'm Gary Chin, an experienced lead software engineer with many years of experience designing, developing, and delivering complex solutions using the Microsoft tech stack (C#,SQL server, MVC, EF). I live just outside of Boston MA.
Never used Microsoft Graph...
And looking for a new job opportunity!!!

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @gychin! 👏

**Comment by BrianTJackett** [comment]
Hello everyone! I'm Brian T. Jackett and I am a Principal PM on our Microsoft Graph Customer and Partner Experience (CPx) team.  I manage our Microsoft Graph Technology Adoption Program (TAP) among other programs.  I look forward to collaborating with you in the coming days and helping answer questions or sharing guidance!

**Comment by 17anik** [comment]
Hello everyone! I am Anik Das and software engineer working on microsoft technologies. I have total 3 years of experience in web developmnet and cloud. Hoping to have a great time here to learn Graph Technology.

**Reply by waldekmastykarz** [reply]
Welcome to Hack Together @17anik! 👋

**Comment by sabeehsharif** [comment]
Hi all, I'm Muhammad Sabeeh Microsoft Certified Professional, MCSE and Senior Microsoft Technologies Developer/Consultant. Automating processes using Microsoft **SharePoint, Graph, .net, Azure and Power Platform**.

- Joined this program to see how Graph is making life easier for others. 
- Very happy to meeting you all Graph champs.

**Reply by daisychaussee** [reply]
Welcome!! Thanks for joining the hack 💯

**Reply by waldekmastykarz** [reply]
Great to have you with us @sabeehsharif! 👏

**Comment by daisychaussee** [comment]
Hi everyone! I'm Daisy Chaussee, a Program Manager for Microsoft 365 Advocacy. I'm excited to see what you all build for this hack!!

**Comment by digidol** [comment]
Hello, 

I am a lecturer at Aberystwyth University in Wales, UK. Entering in my own capacity. I have experience of .NET and ASP.NET Core. Interested to explore Graph and see how it can fit into my teaching.

**Reply by waldekmastykarz** [reply]
Pleasure to meet you @digidol!

**Comment by MsDebbie8591** [comment]
Hello Neal,
I would like to learn all that I can about the Power Platform.  After much thought, I beleive it is the key-the application layer-to maximize the possibility of safety we all had before we knew anything about the internet.  In other words, the application layer of the OSI Model is the weakest link in the OSI model...
What do you think, group? Neal?
Debbie Talley
PS
I am learning Python, Perl, PowerShell, on and on.
________________________________

**Reply by waldekmastykarz** [reply]
Great to have you with us @MsDebbie8591! In this hack we'll be focusing on .NET and Microsoft Graph. That said, Power Platform offers great opportunities for sure and there's a lot of room to integrate low-code Power Platform solutions with pro-code solutions built with .NET.

**Comment by jaymichaud** [comment]
Hi everyone. My name is Jay Michaud, and I work in higher education as a system administrator. I spent the pandemic earning several Microsoft cloud certifications, and I think understanding and using the Microsoft Graph will complement those skills. I have extremely minimal experience with Microsoft Graph and am looking forward to learning!

**Reply by waldekmastykarz** [reply]
Great to meet you @jaymichaud!

**Comment by 0xgurditt** [comment]
Hey everyone, I am Gurditt and I'm a computer science undergrad. I've been trying to learn a lot of new skills recently, and I think GRAPH will be a great addition to those skills. I don't have any experience with it, and I'm really looking forward to learn with all of you!

**Reply by waldekmastykarz** [reply]
Great to have you with us @gurdittcancode! You're absolutely right about Microsoft Graph! Looking forward to hearing how you like it!

**Comment by kamalshree** [comment]
Hi everyone, I am Kamal Shree, a Microsoft 365 Advocacy Developer Advocate. Looking forward to the innovative solutions this hack will bring.

**Comment by maisarissi** [comment]
Hello everyone :)

I'm Maísa Rissi, I'm a Product Manager on our Microsoft Graph Developer Experience (DevX) Team focusing on Microsoft Graph client libraries used by developers when connecting to Microsoft Graph API :)
I look forward to meeting with you in the coming days. Don't hesitate to reach out!

**Comment by SavasCelik** [comment]
Hi everyone, I'm Savas Celik and I'm a software engineer programming ASP.NET Core MVC applications. I saw the banner on https://learn.microsoft.com/en-us/dotnet today and thought this would be a great opportunity to learn about Microsoft Graph.

**Reply by waldekmastykarz** [reply]
Great to have you with us @SavasCelik!

**Comment by johnW-ret** [comment]
Hi all!

I'm a senior Software Engineering student who loves Blazor server and WASM! I have a project using Azure AD and another using Azure AD B2C, but I have yet to do much at all with Graph. I'd like to learn what benefits can be gained from using Graph in my existing AD application, as well as build something new to make me more productive! Nice to meet you all!

I am used to working by myself, but I'd be open to forming a team as well.

**Reply by waldekmastykarz** [reply]
Welcome to the hack @johnW-ret!

**Comment by kc-clintone** [comment]
Hello everyone 👋 

My name is Clinton, a Computer Science student looking to expand my knowledge in more technical fields as possible. I've done projects with MS Graph and Azure AD before and hoping to make the most out of this #HackTogether. 
Nice to meet you all. 😊

**Reply by waldekmastykarz** [reply]
Hey @XtremeArt-KC 👋 Awesome to have you with us!

**Comment by CarnegieJ** [comment]
My name is Carnegie Johnson from southeastern USA. I am a long-time developer, consultant, and trainer at IAYF Consulting where we provide technology focused consultative services. Microsoft 365 is a most valuable product in our catalog of solution services. I look forward to learning from others and to learn by doing while we deep dive into Microsoft Graph and Microsoft 365 during this hackathon. I am a hackathon enthusiast and I love having fun, meeting other participants, and software coding challenges. Best wishes to everyone and I look forward to this exciting event. Cheers! :-)

**Reply by waldekmastykarz** [reply]
Great to meet you @CarnegieJ! I hope you'll enjoy the event and please, don't hesitate to reach out if you have any questions.

**Comment by csc530** [comment]
Hi everyone!
I'm Christofer a current 3rd-year comp sci student in Canada.
This will be the first hack together or similar event I'll be joining. I'm a little nervous/anxious joining but I hope to get a lot out of it. I have worked with .net before but not Microsoft graph, I hope to learn more about them both and improve my skills!
😄

**Reply by waldekmastykarz** [reply]
Hey @csc530! Great to meet you and awesome to have you with us! Please, don't hesitate to reach out if we can help with anything.

**Comment by Geass2116** [comment]
Hello Everyone!
I'm Parth Gautam, 2nd-  Year computer science student in India.
I am Backend developer with knowledge of Node.js, Java and Spring-frameworks and I have basic knowledge of C++ as well.
This is my first 'hack-together' and looking forward to learn new technologies, actually i never worked with .NET or Microsoft graph so instead of winning this hack-together I am more excited to learn new stuff..

Thanks to all Microsoft employees who are with us and who will teach us...🙏

**Reply by 0xgurditt** [reply]
Hi Parth, what college are you from? I'm from India as well!

**Reply by Geass2116** [reply]
Hello Gurditt,
I am from Skit college Jaipur,Rajasthan.

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @Geass2116! 👋

**Comment by aunjum** [comment]
Hello good people 👋
My name is Tanvir Aunjum, I'm a Software Engineer at Ultra-X BD. I love .NET and very excited to work on MS Graph SDK. I would love to focus more on learning then winning. Wish me luck 😊

**Reply by waldekmastykarz** [reply]
Hey @aunjum! Great to have you with us!

**Comment by OzBob** [comment]
Hello all, (@OzBobWA)[https://twitter.com/OzBobWA] Australian Developer here, excited to try MSGraph for the first time.
I have been developing web and server solutions for decades and on Azure recently

Is here a good forum to post onboarding etc. questions?

A note, the [onboarding instructions](https://github.com/microsoft/hack-together/blob/main/setup.md#1---prepare-your-microsoft-365-tenant) refer to signup for the M365 Developer Program, but the emphasis needs to be on existing Visual Studio Ent/Pro subscribers need to use their m365 benefit, I created a new acount and it did not work:

Logging on to the azure portal with the new AAD account , leads me to openning the az CLI, but that needs a storage account which needs an Azure subscription, when prompted to create one the message is: "ozbob@****.onmicrosoft.com is not eligible to sign up for an Azure account."

**Reply by aycabas** [reply]
Hi @OzBob, welcome to Hack Together! We are so excited to have you here. 

We recommend M365 Developer Program account for testing Microsoft Graph integrations. When you build an app with Microsoft Graph, you'll have to login with an M365 account and grant required permissions. In such cases, we recommend you to use M365 Developer Program account. 

Regarding the app development phase using Visual Studio and Azure, you may continue using your own account for the development. M365 Developer Program accounts are only for testing purposes. 

Please let me know if you need more clarification.

**Reply by OzBob** [reply]
Hi @aycabas , I hope no one else has this issue, but just in case I've recorded my experience for others:

After enrolling in the M365 Developer Program a new m365 tenant and AAD are created. I think the missing set of instructions are how to link 'my' azure subscription to the new m365 tenant.  I was able to complete this when I did not use the m365 developer program tenant, but used the one created with my Visual Studio Professional benefits.

**Comment by codewired** [comment]
Hi guys, I am Frank Edu. I live in New York, and I am from Queens. I am a veteran Full Stack Engineer and I have been using .NET since Framework 2.0. I am excited to show Microsoft how to stretch the limits of the Graph SDK. Let the games begin!

**Reply by waldekmastykarz** [reply]
Hey @codewired! 👋  Great to have you with us!

**Comment by jimxxmy** [comment]
Hey everyone! I am Jimit and I am a Software Engineer focusing on .NET. It would be my first time working on Microsoft Graph so looking forward to learning and experience lots from this hackathon.

**Reply by waldekmastykarz** [reply]
Welcome to the hack @jimxxmy! Please, don't hesitate to reach out if you have any questions.

**Comment by Moataz111** [comment]
Hey there, fashionably late to the party, Moataz! 😁 A computer engineer who just recently started learning .NET MAUI and is looking forward to learning here too!

I'm excited to join this hackathon with everyone. Hopefully, I'm gonna have fun learning all the coding skills and some dad jokes along the code lines.

**Reply by waldekmastykarz** [reply]
Right on time! Awesome to have you with us, @Moataz111!

**Comment by MarcosChaarBR** [comment]
Hello Everyone !

I am Marcos Chaar and I'm from Brazil ! I'm Full Stack Developer and I want to learn modern application architectures and integration with Microsoft products. Good studies to all !

**Reply by waldekmastykarz** [reply]
Welcome to Hack Together @MarcosChaarBR! 👋

**Comment by patrickudo2004** [comment]
Hi everyone. stumbled on this, might as well give a go. nothing to lose, a lot to gain.

**Reply by waldekmastykarz** [reply]
That's the spirit! 💪 Awesome to have you with us!

**Comment by SundewBry** [comment]
Hello everyone! 
I am a Software Engineering major graduating in May. I found about about this hackathon when looking at .NET documentation for my senior design project. I haven't worked with these specific .NET tools before, but I'm excited to give it a try!

**Reply by waldekmastykarz** [reply]
Awesome! Great to have you with us!

**Comment by sanjeev40084** [comment]
Hi everyone,
This is sanjeev. Excited for this hackaton. Looking forward to learn and share on Microsoft Graph and .net. 👍

**Reply by waldekmastykarz** [reply]
Hey @sanjeev40084! Great to have you with us!

**Comment by GlebStar1** [comment]
Hi All!
I am Security Engineer with .NET development background! Very passionate about Azure and security! 

P.S. @aycabas @waldekmastykarz
Great intro session! Thanks!

**Reply by waldekmastykarz** [reply]
Thank you for the kind words! Awesome to have you with us @GlebStar1! 👍

**Comment by haakonhelmenrusas** [comment]
Hi folks!
I'm a Developer from Oslo, Norway with experience working on web, Microsoft 365 and Azure. Looking forward to hack together :)

**Reply by waldekmastykarz** [reply]
Hey @Aaronshades! Welcome to the hack! 👋

**Comment by csehammad** [comment]
Hey everyone! My name is Hammad and I'm excited to be participating in this hackathon with all of you. Over the past 14 years, I've had the opportunity to work extensively with the Microsoft technology stack, building a range of enterprise-grade applications.

I'm really looking forward to exploring Microsoft Graph during this hackathon and seeing what we can create with it. It's always great to collaborate with other talented developers and learn new skills.

When I'm not coding, I enjoy reading, and I'm always up for discussing the latest tech trends and innovations. I'm thrilled to be part of this community and can't wait to see what we can achieve together. Let's get hacking!

**Reply by waldekmastykarz** [reply]
Great to meet you @csehammad! Looking forward to seeing your hack!

**Comment by MarvinBangert** [comment]
Hi everyone! my name is Marvin Bangert and I'm a Cloud Architect for Modern Collaboration and Power Platform at glueckkanja-gab in Germany. Microsoft Graph is part of my everyday business within Power Platform, but I would call myself a beginner in .NET, so I am extremely excited on learning new things and make learning my priority for this Hackathon 😊

**Reply by waldekmastykarz** [reply]
Welcome @MarvinBangert! Great to have you with us! Looking forward to your hack!

**Comment by jeffdalgliesh** [comment]
Hello world, I started coding on a [commodore 64](https://en.wikipedia.org/wiki/Commodore_64) in the 80's, got a computer science degree in the 90's (and my microsoft certified developer certificate) then I spent the next 20 years at Chevron working on really cool technology projects and living all over the world. After [Chevron](http://www.chevron.com) I helped start a knowledge graph company ([Maana](https://www.crunchbase.com/organization/maana)) spent 5 years there, created a product called ([WellLine](https://www.welline.com/)) . Then helped start another company ([Geminos](https://www.geminos.ai/)) that creates a causal reasoning modelling tools. Now I am helping to start another company called [Data^2](https://data2.au/) where I am building a natural language processing application for oil and gas operations and asset maintenance. Currently living in Maplebay, BC, Canada. I wanted to do this hackathon to relearn the MS ecosystem of tooling, it seems a lot has changed since visual basic and .Net 1.0. Looking forward to seeing what everyone comes up with :)


01101000 01100101 01101100 01101100 01101111 00100000 01110111 01101111 01110010 01101100 01100100

**Reply by waldekmastykarz** [reply]
01001101 01101001 01100011 01110010 01101111 01110011 01101111 01100110 01110100 00100000 01000111 01110010 01100001 01110000 01101000 00100000 11110000 10011111 10010001 10001011

**Reply by OzBob** [reply]
Microsoft Graph 🍋 in binary, nice!

**Comment by hackemateninja** [comment]
Hi, I am a UX-UI developer with a couple of days of falling in love with F#, and let's try to do something Chido. 🔹👾🧙🏻‍♂️

**Reply by waldekmastykarz** [reply]
Hey @hackemateninja! Looking forward to seeing your hack!

**Comment by HenaGit** [comment]
Hello everyone,
I am Henok from Ethiopia. I am .NET developer and excited to learn Microsoft Graph. Regarding M365 Developer Program account, is it possible to create one with my Microsoft learn account because I don't have a work or school account?

**Reply by waldekmastykarz** [reply]
Hey @HenaGit! Yes, you can sign in to the M365 Developer Program using the same account that you use on Microsoft Learn

**Comment by abaraja-masters** [comment]
Hello Everyone,
I am a Full-Stack Developer with over 6 years of experience. Over the years I have learned many programming languages and frameworks, but have not touched Microsft C#, .Net, or any of its web services. I have recently picked up learning C# and have encountered this hackathon, I am excited to continue learning, practicing, and demonstrating my skills into this fun project.

**Reply by waldekmastykarz** [reply]
Welcome to the hack @abaraja-masters! Great to have you with us!

**Comment by Ahuoyiza** [comment]
Hi everyone, I'm Sophia a Software engineer.
My primary stacks are typescript, javascript and.net.
Haven't used graph before and hope to do that during this hack!
Lots of learning in store for me in the next few days i guess😀

**Reply by waldekmastykarz** [reply]
Hey @Ahuoyiza! Great to have you with us! Looking forward to seeing your hack!

**Comment by EmreAka** [comment]
Hello, My name is Emre. I'm a self-taught .NET developer.

**Reply by aycabas** [reply]
Aramiza hosgeldin @EmreAka 😊

**Reply by waldekmastykarz** [reply]
Welcome the hack @EmreAka!

**Comment by ta-chi-na** [comment]
HI FRIENDS!! My name is Tachina. I am a student learning code all around the world. I can't wait to learn new hacks during this period! You all inspire me to learn every day and strengthen my code. Good luck to you all and keep learning/educating. ☮️ .

**Reply by waldekmastykarz** [reply]
Hey @ta-chi-na! Pleasure to meet you!

**Comment by shariharan1** [comment]
Hello everyone! I'm a programmer with over 25 years and new to MS Graph; Hopefully can get good exposure on this platform. Wishing all a good hackathon experience. Cheers.

**Reply by aycabas** [reply]
Welcome @shariharan1, we are so excited to have you here 🤗

**Reply by waldekmastykarz** [reply]
Welcome to the hack @shariharan1!

**Comment by alexrbranch** [comment]
Hey everyone! I am a sophomore in college just starting to learn C# and .NET! Look forward to learning more!

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @alexrbranch!

**Comment by abhipray-chavan** [comment]
Hello Everyone! My name is Abhipray. I am a final year student and am new to C# and .NET . Looking forward to a great learning experience.

**Reply by waldekmastykarz** [reply]
Welcome @abhipray-chavan! Great to have you with us!

**Comment by svrooij** [comment]
Hi everybody,

I'm Stephan van Rooij a Microsoft MVP in security and I'll be joining to do some fun stuff with the Graph API and am available for answering questions during this #hackathon. Have fun, and lets build awesome stuff together!! 

If you're at the point where you build something cool, be sure to add a **README.md** file to the project. I think every project needs badge, so be sure to check my [every project needs a badge](https://svrooij.io/2023/01/02/badges-badges-badges/) post on how to do just that.

And by the way, I'm still looking for some teammates that want to help out making my project awesome.

**Reply by waldekmastykarz** [reply]
Awesome to see you here @svrooij! Looking forward to seeing your hack!

**Comment by anish1617** [comment]
Hello everyone! I'm Anish Mandal, working as a Jr. C# .NET developer. I would like to enhance my skills alongside the fellow passionate developers and build the network.  Hoping to gain experience and grow skills in .NET and MsGraph.

**Reply by waldekmastykarz** [reply]
Welcome to the hack @anish1617! Great to have you with us!

**Comment by AniniBTC** [comment]
Hi I'm a novice here
Any one here to teach me something new

**Reply by waldekmastykarz** [reply]
You're in the right place @AniniBTC! Welcome to Hack Together! 👋

**Reply by AniniBTC** [reply]
Any new friend ?

**Comment by AniniBTC** [comment]
Thanks

**Comment by nojronatron** [comment]
Hello everyone! My name is Jon and I am a full stack software developer. I enjoy working with Java, C#, and the MERN stack. I have experience as a software tester and a systems administrator, and am working toward earning my first position as a software developer. I look forward to taking on this challenge with MS Graph!

**Reply by waldekmastykarz** [reply]
Great to have you with us, @nojronatron! 👋

**Comment by godfreyowidi** [comment]
maybe I should keep it simple and call my app something like "stupid potato...". want to avoid rabbit hole overthinking and just justify the concepts..simple and easy :)

**Comment by manoj20decpatel** [comment]
Hi All,
I am Manoj Kumar Patel, Sr. Integration Specialist at Hitachi Solutions, I have 12yrs of experience working with Microsoft technologies. I see a lot has changed over past 3yrs with Graph Api, the last time I utilized graph explorer for a SharePoint integration work in 2020 with a very little info in public domain, today Graph APi has grown out to be a BIG giant and future torch bearer for all Microsoft product integrations.

**Reply by waldekmastykarz** [reply]
Welcome to the hack @manoj20decpatel! Great to have you with us!

**Comment by KamilSiebyla** [comment]
Hi folks, 

my name is Kamil. I am a Software Developer with .NET background. I am here to learn something new and play around with graph. I hope everyone will have fun during creation of their apps :-). 

Thanks and have a nice evening/day :-) 

/Kamil

**Reply by waldekmastykarz** [reply]
Great to meet you @KamilSiebyla! Welcome to the hack!

**Comment by K8Liu** [comment]
Hello Everyone!

My name is Kevin, I'm a software engineer in Seattle, Washington. I am joining the hackathon with my friends, and we are all really excited to learn about .NET and to utilize Microsoft Graph in our project. I look forward to meeting everyone and learning something new. 

Kevin

**Reply by EmreAka** [reply]
Good luck!

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @K8Liu! Looking forward to seeing your hack!

**Comment by diannedejesus** [comment]
Hello everyone, my name is Dianne.  I have been getting back into development for the past two years. I already was playing around with graph with Nodejs and decided to use this hackathon to try making a desktop app just for fun. Excited to see what everyone comes up with.

**Reply by waldekmastykarz** [reply]
Great to have you with us @diannedejesus! Looking forward to seeing your hack!

**Comment by asieradzk** [comment]
Hi Everyone,
I am Adrian, grad student in biochemical engineering and side-hustler .net dev. I am looking forward to seeing your projects at the end.
🚀🚀🚀

**Reply by waldekmastykarz** [reply]
Hey @BlueKiwiEntertainment! Great to have you with us!

**Comment by ehafenmaier** [comment]
Hello Everyone!

My name is Erich and I've been working with .NET since as far back as I can remember. Started with ASP.NET Web Pages (actually started with just ASP but we're focused on .NET here 😉), then migrated to ASP.NET MVC applications and now I'm all in with .NET 5 (.NET Core) and beyond! I'm currently working towards my Azure Developer Associate certification and this hackathon seems like a fun thing to do that would also add to my Azure education.

**Reply by waldekmastykarz** [reply]
Great to meet you @ehafenmaier! Welcome to the hack! 👋

**Comment by blackhawk64** [comment]
Hello friends!

My name is Alexis, from Mexico and I am a Microsoft enthusiast since a couple years ago hehe. I worked at a company that sells Microsoft solutions so I started working with C#, .Net, Azure and so on; it was a really good experience to been working on Microsoft based applications. Nowadays I still programming with .Net and other technologies (I'm a fullstack dev), but my actuall job doesn't requires me to use all Microsoft enviroment, so I thought it would be awesome to get envolved again with this things by myself. I really like to learn new things and this will add greats experiencies to my skillset 🥳

**Reply by waldekmastykarz** [reply]
Excellent choice! Great to have you with us @blackhawk64! 👋

**Comment by Vedshruti-11** [comment]
Hi, I'm Vedshruti Sarkar, a Computer Science Engineering student in my final year. I have recently started working with .NET and C#. I have knowledge in Java, C++ and MERN stack. Hope to learn and experience something new building web app with Graph and .NET !

**Reply by waldekmastykarz** [reply]
Great to have you with us @Vedshruti-11!

**Comment by justinyoo** [comment]
Hey everyone! I'm Justin Yoo, Senior Cloud Advocate at Microsoft focusing on .NET and Azure. I'm happy to be here and can't wait to see all of you on the Learn Live show on March 8th (PST)!

**Reply by waldekmastykarz** [reply]
👋

**Comment by tajbender** [comment]
Hi everyone!

My name is Thorsten Jung from Franconia, Germany. Born 1976, my first mouse held in hands was a commodore amiga mouse ✋ Being a child of the 90s, I took the hard way of learning MS-BASIC on amiga before learning 68k assembler for performance reasons 😁 

Got my first Wintel machine in '97 while graduating from high school. From 2k on, I worked as an developer at a big company in Hamburg, Germany. There I made especially B2B solutions for the enterprise, while using Oracle DB, Unix Shell _("U weren't supposed to be here, u know?" - "Hello Bash, nice to see u again")_ and C/C++ for mainframes.

After some years, I got a burnout cause of manic depression, so I'm retired now. I love cats, music, xbox and hacking the world, cause coding was always my passion, not just a job. So finally, I work as a freelancer now, interested in web design, .net, windows, guis and stuff like that.

When time comes, I will create a Turrican like side scroller 🙌 🥇 In the meantime, I'm coding the electrifier project, that was halted several times due to my health condition.

Btw. I love GitHub. It's like facebook for passionate coders. Thx to everyone @microsoftopensource for making this lovely place possible, and even better every then and when.

Thanks for reading, 
regards, 
tajbender

p.s.: feel free to ask

**Reply by waldekmastykarz** [reply]
Great to meet you @tajbender! Welcome to Hack Together!

**Comment by kusscaroline** [comment]
Hi Everyone!
My name is Caroline, am from London. I am currently writing a set of books for newbies- that was how I came across this challenge. So, here I am; I 've got to win the money, because I need it to continue paying for my Azure after free period, hehehe.
Any-hoh - what I am guessing here, is that the main aim of this project (after going round-and-round) is that you guys are trying to get people to stop using Azure AD Graph and to start using Microsoft Graph?

**Reply by waldekmastykarz** [reply]
Hey @kusscaroline, great to have you with us! The goal of this hackathon is to get people experience building apps using .NET and Microsoft Graph. Microsoft Graph offers you access to many types of data stored on Microsoft 365. Check out https://graph.microsoft.com/ to learn more about what's possible. Looking forward to seeing your hack!

**Comment by tarzzi** [comment]
Hi all!
I'm Tarmo from Finland, currently working as an developer for SharePoint (SPFx), Power Platform and different intranet solutions.
Last written C# lines were three years ago at IT-studies, so I'm here to get my C# and .NET skills back on track (or at least so that I don't forget everything totally... 😄)
Happy hacking y'all! 👏🥳

**Reply by waldekmastykarz** [reply]
Welcome to the hack @tarzzi! Great to have you with us! 👏

**Comment by YusC3** [comment]
Hello,
From the US hacking with a friend. Previously employed at MS, I wanted to participate in a hack culture I know I enjoy and well as create something with MS Graph. Excited!

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @YusC3. Looking forward to seeing what you'll hack!

**Comment by guanryan1234** [comment]
Hello, 

My name is Ryan, I am an elementary programmer doing my first hackathon with a friend. Excited to meet everyone and learn something new!

**Reply by waldekmastykarz** [reply]
👋  Hey @guanryan1234! Great to have you with us!

**Comment by dmasuw** [comment]
Hello everyone, my name is Dima, I'm a software developer at UW. Just started working with MS graph. Very happy to participate in this hackathon and learn more.

**Reply by waldekmastykarz** [reply]
Hello @dmasuw! Great to have you with us! 👏

**Comment by svetlanamoscalenco** [comment]
Hello Everyone👋,
 
My name is Svetlana. I am a Software Developer with .NET background. I am excited about this hackathon, an excellent opportunity to learn Microsoft Graph.
Looking forward to hacking together!!💪👍

**Reply by waldekmastykarz** [reply]
Welcome to Hack Together @svetlanamoscalenco! Great to have you with us!

**Comment by fenildesai** [comment]
Hello, I'm Fenil Desai, a bit late to the hack party, still looking forward to learn about Graph API as have been working with .NET for years & would love to utilise the power of Graph API to see what amazing things can be built.

**Reply by waldekmastykarz** [reply]
Welcome @fenildesai! Great to have you with us!

**Comment by sergioocode** [comment]
Hello, how are you all? It's been a year since I discovered Graph and I was already familiarizing myself with the API. now trying to learn sdk v5 Awesome work guys!

**Reply by waldekmastykarz** [reply]
Great to have you with us @sergiokml! 👏

**Comment by castlecreati** [comment]
Hello everyone.
Since the pandemic my job went from being a Training Service in a Professional Institute, to an IT department for which I had to raise an educational tenant and some infrastructure in Azure. I have since tried to learn as much as I can about Azure, Azure AD and Asp.Net Core, although I am still a beginner, and at 55 years old my brain cells are on crutches.

Microsoft Graph may be an essential element in my work. And this hackathon is a great way for me to get started with Blazor and take a look at it.

I have a question regarding the project submission - does midnight on March 15 refer to 00:00 on the 15th, or 24:00 on March 15 (or 00:00 on March 16)?

**Reply by waldekmastykarz** [reply]
Great to have you with us @castlecreati! You've got the whole day of March 15 to hack and submit your hack by the end of day Wed, Mar 15.

**Reply by castlecreati** [reply]
Ok, .... Thank you !!!

**Comment by SomebodyToLove1337** [comment]
Hello all,

I have missed to introduce myself until now.
My name is Benedikt and I am really happy to participate in my first coding challenge. I've been in business as an IT administrator for about 20 years. But I haven't really been able to program yet, I've been working with scripts in Batch and PowerShell and a bit of webdesign.
But in the meantime I finished my first console app in C# and hope I manage to submit it tonight. I keep my fingers crossed that it will make it to the jury.

Unfortunately the challenge will be over soon.
But I really enjoyed the hack-together and I will continue to develop the app afterwards.

In any case, I'll stay on the ball with C# and .NET, even if I don't understand that much yet I have 1,000 ideas what I still want to implement with it. And since I've been working with Office 365 since 2010, Microsoft Graph will definitely play a big role in my applications.

Many thanks to the organizers.

Best regards
Benedikt

**Reply by aycabas** [reply]
Hi @SomebodyToLove1337, It's a pleasure having you in the challenge! Looking forward to seeing what you've built! 😊

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @SomebodyToLove1337! Looking forward to seeing your hack!

**Reply by SomebodyToLove1337** [reply]
Thanks,

finally we submit our Project!!!!! 👍

**Comment by Waraemon** [comment]
Hello everyone,

I forgot to connect until now.

I am a C# Developer in the banking industry. My main work is solo tasks with M365 development in an internal system using MS Graph API. I am happy to share my experiment with the community.

This is the first time that I have joined a hackathon, just taken part in CTFs before because of the security heavy in the Bank.

Many thanks to the organizers and this hack make me don't alone!

Best Regard,

**Reply by waldekmastykarz** [reply]
Awesome to have you with us @NagaDemon!
